omnibus/config/software/cleanup.rb

```diff
@@ -0,0 +1,8 @@
+name 'cleanup'
+default_version '1.0.0'
+skip_transitive_dependency_licensing true
+license :project_license
+
+build do
+  delete "#{install_dir}/embedded/nodejs"
```

Comment:
Yes, I believe it does.  I am unable to find any node_modules folder in the final package